programs/wen_transfer_guard/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "wen_transfer_guard"
+version = "0.1.0"
+description = "Created with Anchor"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "lib"]
+name = "wen_transfer_guard"
+
+[features]
+default = []
+cpi = ["no-entrypoint"]
+no-entrypoint = []
+no-idl = []
+no-log-ix-name = []
+idl-build = ["anchor-lang/idl-build", "anchor-spl/idl-build"]
+
+[dependencies]
+anchor-lang = { version = "0.30.0", features = ["interface-instructions"] }
+anchor-spl = "0.30.0"
+spl-tlv-account-resolution = "0.4.0"
+spl-transfer-hook-interface = "0.3.0"
```

Comment:
This should be referenced from workspace. We'll try to keep all these programs on the same version of these packages.